csharp/ql/src/semmle/code/csharp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -245,15 +249,6 @@ newtype TTranslatedElement =
     not ignoreExpr(initList) and
     exists(initList.getElement(elementIndex))
   } or
-  // The value initialization of a range of array elements that were omitted
```

Comment:
In cpp this type of array initialization was used when only a part of the array was explicitly initialized, and the rest was compiler initialized with the default value. Since in C# everything needs to be initialized, I did not see any use of this type of initialization.

Reply:
OK, good to know.